src/StartProject.Asp/Config/Moryx.Products.Model.ProductsContext.DbConfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "ConfigState": "Generated",
-  "Host": "localhost",
-  "Port": 5432,
-  "Database": "Framework6_ProductsContext",
-  "Username": "postgres",
-  "Password": "postgres"
+  "ConnectionSettings": {
```

Comment:
I like that we habe one db config for sqlite and one for postgres by default. However, please rename the database to Framework6_ProductsContext. 
And is there a reason you deleted all the other configs that were checked in?

Reply:
i changed the configs a lot , so i don't even remember what was there initially.